Src/java/cql-to-elm/src/main/java/org/cqframework/cql/cql2elm/Cql2ElmVisitor.java

```diff
@@ -2786,7 +2786,14 @@ public Object visitSetAggregateExpressionTerm(cqlParser.SetAggregateExpressionTe
         if (ctx.dateTimePrecision() != null) {
             per = libraryBuilder.createQuantity(BigDecimal.valueOf(1.0), parseString(ctx.dateTimePrecision()));
         } else if (ctx.expression().size() > 1) {
-            per = parseExpression(ctx.expression(1));
+            var perExpression = parseExpression(ctx.expression(1));
+            // Hack to get the per value from a literal
```

Comment:
```suggestion
            // Implicitly convert a literal to a quantity here
            // Note that although this is declared as a conversion, the translator doesn't pick it up because it
            // won't instantiate the generic signature because generic signature instantiation logic is not considering
            // implicit conversions to class types.
```